### DISCUSSION TITLE
Separate style manager

### DISCUSSION THREAD

**Discussion by khanhleemtp** [GrapesJS/grapesjs#4290] [Q&A]
i want to separate style in two part. Advance and Normal. How to separate style manager

**Comment by artf** [comment]
You can replace the default Style Manager UI with your own implementation https://grapesjs.com/docs/modules/Style-manager.html#custom-style-manager

**Comment by Ju99ernaut** [comment]
I've separated them by defining different Style Manager configs for each section, then switching between them when a tab is selected:

```js
switch (type) {
    case 'basic':
        stm.getSectors().reset(basicSector);
        break;
    case 'advanced':
        stm.getSectors().reset(advancedSector);
        break;
    default:
        break;
}
editor.trigger('component:toggled');
```

`stm` is `const stm = editor.Styles`;
